sections/09_SmartEconomy.md

```diff
@@ -0,0 +1 @@
+# Towards the Smart Economy: the three pillars of NEO
```

Comment:
This should not be included in specification.

Reply:
Are you sure?
The specification could have this description about NEO foundation and long-term plans to accomplish its design.

It could be a kind of Economical Specification.

Reply:
I'm sure that the specification is used to describe the technical details of NEO.

Reply:
I think this part should be included in the white paper, not in the specification.